Ginger/GingerCoreCommon/VariablesLib/VariableSelectionList.cs

```diff
@@ -121,14 +124,53 @@ public override void ResetValue()
         {
             if (OptionalValuesList.Count > 0)
             {
+                OptionalValuesList.CurrentItem = OptionalValuesList.ElementAt<OptionalValue>(0);
```

Comment:
no need this, let's find the index of current value by the value itself like optionalValuesList.IndexOf(Value)